docker-compose.yaml

```diff
@@ -325,7 +325,7 @@ services:
       - ./scripts/docker-compose/volume-data/openklant-database-data:/var/lib/postgresql/data
 
   openklant.local:
-    image: docker.io/maykinmedia/open-klant:2.1.0
+    image: docker.io/maykinmedia/open-klant:2.3.0
```

Comment:
I think we should let this be determined by Dimpact.
We will have to determine how we are going to deal with this in the context of multiple versions.... Long running branches? Ability to support multiple versions, or how?